src/app/test-suites/page.md

```diff
@@ -2,19 +2,30 @@
 title: Test Suites Overview
 description: Replay Test Suites bring time-travelling DevTools to your existing Cypress, Playwright or any other test suites
 ---
+
 Replay Test Suites bring **time-travelling DevTools to your test runs**. You’ll get a better insight into how your test script interacts with application under test. Besides zooming in on your test execution, you can zoom out to get a birds-eye view of your teste executions and **drive down test failures and flakes**.
 
 {% basic icon="treeview" %}
+
 ## Test Steps Trace
+
 Replay Devtools work great with your existing test suite. Cypress and Playwright plugins allow you to **rewind or fast forward** to any command from your test run.
 
 **Command details** contain information on passed arguments, returned values and targeted element. Commands are integrated with the rest of DevTools. This means you can jump from a command detail into [Elements panel](/time-travel/elements-panel), or inspect API calls in the [Network panel](/time-travel/network-panel).
 {% /basic %}
 
-![Playwright panel](/images/playwright_panel.png){% class="bg-gradient-to-tr from-blue-200 via-fuchsia-300 to-orange-400 " %}
+{% figure
+    alt="Playwright panel"
+    src="/images/playwright_panel.png"
+    gradient="bg-gradient-to-tr from-blue-200 via-fuchsia-300 to-orange-400"
+    height=370
+    width=640
+/%}
```

Comment:
Setting the height and width ensures that the image scales smoothly and retains the same gradient border size.

It can work without explicit size provided dimensions are set on the container via classes but this approach can't guarantee the border is the same (without measuring on the client)

```
{% figure
    alt="Playwright panel"
    src="/images/playwright_panel.png"
    gradient="bg-gradient-to-tr from-blue-200 via-fuchsia-300 to-orange-400"
    class="h-64 w-96"
/%}
```